src/components/SelectionList/UserListItem.tsx

```diff
@@ -18,45 +29,75 @@ function UserListItem({item, textStyles, alternateTextStyles, showTooltip, style
     const hoveredBackgroundColor = !!styles.sidebarLinkHover && 'backgroundColor' in styles.sidebarLinkHover ? styles.sidebarLinkHover.backgroundColor : theme.sidebar;
 
     return (
-        <>
-            {!!item.icons && (
+        <BaseListItem
+            item={item}
+            wrapperStyle={[styles.flex1, styles.justifyContentBetween, styles.sidebarLinkInner, styles.userSelectNone, styles.peopleRow, isFocused && styles.sidebarLinkActive]}
+            selectMultipleStyle={[
+                StyleUtils.getCheckboxContainerStyle(20),
+                styles.mr3,
+                item.isSelected && styles.checkedContainer,
+                item.isSelected && styles.borderColorFocus,
+                item.isDisabled && styles.cursorDisabled,
+                item.isDisabled && styles.buttonOpacityDisabled,
```

Comment:
same here, the same styles are in RadioListItem, could be moved to StyleUtils

Reply:
done